frontend/src/__mocks__/mockModelVersionList.ts

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable camelcase */
 import { ModelVersionList } from '~/concepts/modelRegistry/types';
-import { mockModelVersion } from './mockModelVersion';
 
-export const mockModelVersionList = (): ModelVersionList => ({
-  items: [mockModelVersion({ author: 'Author 1', registeredModelId: '1' })],
+export const mockModelVersionList = ({
+  items = [],
+}: Partial<ModelVersionList>): ModelVersionList => ({
+  items,
   nextPageToken: '',
   pageSize: 0,
-  size: 1,
+  size: 2,
```

Comment:
make the size dynamic
```suggestion
  size: items.length,
```